parsec/core/gui/users_widget.py

```diff
@@ -257,6 +257,7 @@ def invite_user(self):
         if not user_email:
             return
 
+        self._user_email = user_email
```

Comment:
It is better to get the email from the return value of `_do_invite_user` rather than store it as an attribute. Storing this kind of attributes can lead to a lot of weird errors later on.

Reply:
Done